### PR TITLE
remove cluster token in sentry

### PR DESCRIPTION
**Pull request by BumbleFeng**
Signed-off-by: Feng Huang <fehuang@redhat.com>

### CODE REVIEW

File: reconcile/cli.py [hunk 1]
```diff
@@ -102,9 +103,21 @@
 from utils.unleash import get_feature_toggle_state
 
 
+def before_breadcrumb(breadcrumb):
```
Comment (jfchevrette) [inline reconcile/cli.py:106]: Can you add a docstring to explain what this does? I expect we will eventually want to add more things in there to mask other types of sensitive informaton

**Comment by jfchevrette** [comment]
LGTM from a code/sentry sdk standpoint but I can't think of a way we can test this before deploying it.

OTOH, I don't think the code can do harm so +1 to :ship:

**Comment by maorfr** [comment]
LGTM as well
@jmelis PTAL

**Review by maorfr — APPROVED** [review]
this is probably fine to promote and see if it works

**Pull request merged**